packages/noya-renderer/src/components/SketchFileRenderer.tsx

```diff
@@ -407,16 +407,13 @@ export default memo(function SketchFileRenderer() {
       return;
 
     const layer = Layers.access(page, indexPath.indexPath) as PointsLayer;
-    const decodedCurvePoint = Primitives.decodeCurvePoint(
-      layer.points[indexPath.pointIndex],
-      layer.frame,
-    );
 
     return (
       <>
         <PseudoPathLine
           point={interactionState.point}
-          decodedCurvePoint={decodedCurvePoint}
+          curvePoint={layer.points[indexPath.pointIndex]}
+          frame={layer.frame}
```

Comment:
Would it be better to pass in the entire layer and indexPath rather than the curvePoint and frame?